### PR TITLE
Implemented `clickCount` for mouse events

### PR DESCRIPTION
**Pull request by ShaharMS**
`clickCount` seemed to have missed an implementation until now, and i needed the ability to detect click sequences for this PR (#2580 ). This implements clickCount for both HTML and native.

### CODE REVIEW

**Comment by ShaharMS** [comment]
this should be merged before meging this: https://github.com/openfl/lime/pull/1579

File: src/openfl/display/Stage.hx [hunk 1]
```diff
@@ -2611,8 +2613,9 @@ class Stage extends DisplayObjectContainer #if lime implements IModule #end
 			event.localX = local.x;
 			event.localY = local.y;
 			event.target = target;
+			event.clickCount = window.clickCount
```
Comment (player-03) [inline src/openfl/display/Stage.hx]: Missing semicolon.

Comment (ShaharMS) [reply src/openfl/display/Stage.hx]: Ill fix that in a moment. Let's forget that happened 😅

**Comment by Caglar7177** [comment]
> `clickCount` seemed to have missed an implementation until now, and i needed the ability to detect click sequences for this PR (#2580 ). This implements clickCount for both HTML and native.

**Comment by ShaharMS** [comment]
Bump? Or is this irrelevant?

**Pull request merged**